app/models/institution_builder.rb

```diff
@@ -516,19 +515,18 @@ def self.build_zip_code_rates_from_weams(version_number)
       bah,
       dod_bah,
       concat_ws(', ', physical_city, physical_state) as physical_location,
-      v.number,
       #{conn.quote(timestamp)},
       #{conn.quote(timestamp)},
-      v.id
-      FROM weams INNER JOIN versions v ON v.number = ?
+      #{version_id}
+      FROM weams
     WHERE country = 'USA'
       AND bah IS NOT null
       AND dod_bah IS NOT null
```

Comment:
Don't we need to filter by version_id?

Reply:
Do you mean adding something like `AND version_id = #{version_id}`

I don't think that is actually necessary here since we are passing in `version_id` as a parameter here to set it for `zipcode_rates` and `weams` does not have a `version_id`